package.json

```diff
@@ -57,9 +57,7 @@
     "babel-polyfill": "^6.26.0",
     "babel-preset-es2015": "^6.24.1",
     "chai": "^4.1.2",
-    "codecov": "~1.0.1",
```

Comment:
Been a while since I've worked in this repo. When did we stop using codecov locally? 😂 

https://github.com/vega/compassql/pull/280/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R20

Reply:
We handle reporting coverage in github actions now https://github.com/vega/compassql/pull/453/files#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88R39